src/content/docs/merchants/get-started/multistore.mdx

```diff
@@ -10,6 +10,34 @@ import Tasks from '@components/Tasks.astro';
 import Task from '@components/Task.astro';
 import { Steps } from '@astrojs/starlight/components';
 
+
+## What is Multistore?
+
+**Multistore** refers to the ability to run multiple storefronts from a single codebase and content repository. This is useful for:
+
+- **Localization**: Serving different languages and currencies (e.g., English/US, French/Canada) from the same site, with localized content and commerce data.
+- **Multiple Brands or Regions**: Operating several brands or regional storefronts, each with their own configuration, but sharing core code and content.
+
+There are two main approaches to multistore:
+
+1. **Multiple Domains**: Each domain points to a different commerce backend or configuration, but shares the same code and content.
+2. **Subfolders (Root Folders)**: A single domain with subfolders (e.g., `/en/`, `/fr/`) for each locale or store. Each subfolder can have its own localized content and commerce configuration.
+
+**Key features:**
+- **Editorial and Commerce Localization**: Editorial content (pages, blocks) and commerce data (products, prices, currencies) are both localized.
+- **Automated Translation Tools**: Tools are provided to automate translation of content and manage updates across languages.
+- **Configurable Headers**: Storefronts can send different headers to the commerce backend to select the correct store, language, and currency.
+
+---
+
+## How Does Multistore Configuration Work?
+
+- **Config Files**: Each store or locale is defined in configuration files, specifying which headers to send to the commerce backend, which content folders to use, etc.
+- **Store Switcher**: Out-of-the-box, a store switcher UI is provided so users can switch between stores/locales on the frontend.
```

Comment:
It’s not entirely out of the box:
- A store switcher that authors can edit, which includes links to different stores and other content
- Functionality that allows shoppers to switch between stores